README.md

```diff
@@ -14,8 +14,57 @@ the packages individually, for example:
     make -C debs/xenial/archivematica
     make -C debs/xenial/archivematica-storage-service
 
-### Test packages
+#### Using parameters
 
+Most makefiles support parameters in the form of environment variables. They 
+may be a bit different between packages, but the most common are ``BRANCH``,
+``VERSION`` and ``RELEASE``.
```

Comment:
Code spans use a single backtick string instead of two.

> A code span begins with a backtick string and ends with a backtick string of equal length. The contents of the code span are the characters between the two backtick strings, with leading and trailing spaces and line endings removed, and whitespace collapsed to single spaces.